.env

```diff
@@ -1,4 +1,22 @@
-LOCAL_VIVO_HOME=./vivo-home
-RESET_HOME=false
-RESET_CORE=false
-VERBOSE=no
+# Variable substitution for docker-compose.yml
```

Comment:
What do you think about separating the .env file into 2 files:
- .env.solr
- .env.tomcat

Would that make it easier to configure for the end user?

Reply:
The `.env` currently is for substituting variables for the `docker-compose.yml` and done without additional docker compose command arguments. These variables are technically not service environment variables unless used to set under `environment` of each service. If necessary, an environment file per service can be added and used for the environment variables of the service. At this time, I opted against env file per service as there is more build time variables (build args, volume, and port variables) needed than actual service environment variables.

Reply:
I understand, then you should leave it as is. I was just wondering what's your opinion and did my suggestion cross your mind at some point during development 😄.